src/yki/handler/exam_payment_new.clj

```diff
@@ -156,12 +120,12 @@
           (let [from-inclusive     (LocalDate/parse from)
                 to-exclusive       (-> (LocalDate/parse to)
                                        (.plusDays 1))
-                completed-payments (payment-db/get-completed-payments-for-timerange db from-inclusive to-exclusive)]
-            (->> completed-payments
-                 (with-organizer-names url-helper)
-                 (payments-data->csv-input-stream url-helper)
-                 (csv-response {:from from
-                                :to   to})))
+                completed-payments (payment-db/get-completed-payments-for-timerange db from-inclusive to-exclusive)
+                result             (->> completed-payments
+                                        (with-organizer-names url-helper)
```

Comment:
Pitäiskö täällä olla `%` myös lopussa? IDE herjaa `Incorrect arity 1 for ...`

Reply:
Ei pidä, `%` olisi validia syntaksia vain jos se formi mihin se syötetään olisi anonyymifunktio, esim. `#(with-organizer-names url-helper %)`. Tuon IDEn voisi kyllä katsoa kuntoon, se on nyt vaan väärässä.

Reply:
IDE ei ehkä osaa tunnistaa `->>`-makroa? Toi completed-payments siis syötetään rivillä 125 viimeiseen positioon, ja tämän tulos taas seuraavan rivin vikaan positioon, jne. https://clojure.org/guides/threading_macros#thread-last

Reply:
Joo ja ei näytä mun IDE osaavan myöskään käsitellä `->` makroa oikein sillä jos vaihdan tuolle riville 124 `->> muotoon ->` herjaa edelleen samasta asiasta.